src/styles/blocks/stars.css

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
+  background-image: url(../../images/star.svg);
+  margin: 2px 3.4px;
+}
+
+.stars__star:first-child {
+  margin-left: 2px;
+}
+
+.stars__star:last-child {
+  margin-right: 2px;
+}
+
+.stars__star:nth-child(3) {
+  margin-right: 4px;
+}
```

Comment:
The distance between all stars is the same

use margin-right: 4px to add space between stars and remove margin-right from the last stars in a row using `:last-child `rule